src/components/modal/__tests__/Modal.test.js

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import { shallow } from "enzyme";
+import "jest-extended";
+import sinon from "sinon";
+
+import Modal from "../Modal";
+
+describe("Modal", () => {
+  it("should render correctly with default state", () => {
+    const props = {
+      show: false,
+      children: <div />,
+      onClose: () => {}
+    };
+    const component = shallow(<Modal {...props} />);
+    expect(component).toMatchSnapshot();
+  });
+});
+
+describe("Modal render correct show prop", () => {
+  const props = {
+    show: false,
+    children: <div />,
+    onClose: () => {}
+  };
+  const component = shallow(<Modal {...props} />);
+  it("check proptype", () => {
+    expect(component.hasClass("hidden")).toEqual(true);
+  });
+
+  it("check proptype", () => {
```

Comment:
El texto dentro del it, debería de describir que hace el test: https://medium.com/@yahelyechieli/5-tips-for-writing-great-javascript-unit-tests-86296ad2d997